guides/configure-oE-devEnviron-ubuntu-vm/How-to-install-openEQUELLA-Development-Environment-in-Virtual-Box-Ubuntu-OS.md

```diff
@@ -0,0 +1,626 @@
+
+### <a id="user-content-install-sbt" class="anchor" aria-hidden="true" href="#install-sbt"></a>Install SBT
+
+SBT is used to make the project. Because we installed sdkman to install Java, we can also use sdkman to install SBT, follow the commands below.
+
+`$ sdkman install sbt`
+
+### <a id="user-content-create-branch" class="anchor" aria-hidden="true" href="#create-branch"></a>Create a New Branch of openEQUELLA in Git
+
+1. It’s best not to make changes directly to the develop branch. So create a new branch in Git.
+
+Verify you are on currently on the develop branch.
+
+`$ git branch --show-current`
+
+![git ssh](images/clone/05%20git%20ssh.png)
+
+2. Associate the originating openEQUELLA repo with your fork.
+
+`$ git remote add upstream git@github.com:apereo/openEQUELLA.git`
+
+3. Verify you are using SSH and have the upstream repository set.
+
+`$ git remote -v`
+
+![git ssh](images/clone/05%20git%20ssh.png)
+
+4. Make sure your develop branch is up to date.
+
+`$ git pull`</br>
+`$ git push`
+
+5. Create the new branch and switch to the new branch in a single command.
+
+`git checkout -b enhancement1`
+
+![git new branch](images/clone/06%20git%20new%20branch.png)
+
+`git remote set-url origin'</br>
+ 'git@github.com:someone/openEQUELLA.git`
+
+### <a id="user-content-import-sbt" class="anchor" aria-hidden="true" href="#import-sbt"></a>Import sbt project into Intellij
+
+1. An easy way to make sure your dependencies are updated is to open the openEquella Respository as an sbt project. Intellij will prompt you to update your dependencies. To do this launch Intellij and click on “Open or Import.”
+
+![import proj](images/Intellij/01%20import%20proj.png)
+
+2. Navigate to build.sbt and click ok.
+
+![sbt file](images/Intellij/02%20sbt%20file.png)
+
+3. Click “Open as Project”
+
+![open as project](images/Intellij/03%20open%20as%20project.png)
+
+4. If you have messages in the Event Log prompting you to update dependencies, go ahead and update them.
+
+### <a id="user-content-compile-run" class="anchor" aria-hidden="true" href="#compile-run"></a>Compile and Run openEQUELLA Server
+
+1. f you don't have enough RAM on your host OS, please bring your max heap config down to 3072 or even lower to 2048 if sbt fails ( via the -mem in **.sbtopts** ). 
+
+NOTE:  If you cannot cannot see **.sbtopts** from the file manager you can type cntr +H which will reveal hidden files. The other options is to
+ access the file from a terminal windows. Navigate to the openEquella directory and type the command below.
+
+![sbtopts](images/Intellij/04%20sbtopts.png)
+
+2. Either way change the value of –mem to 3072 or 2048 then cntr + o to save changes to the file. Then cntr + x to close the file.
+
+![set mem](images/Intellij/05%20set%20mem.png)
+
+3. cd to the {Equella repo} directory then run the command below.
+
+`$ sbt installerZip`
+
+4. Run the commands in the following order. 
+
+`Sbt:Equella> compile`</br>
+`Sbt:Equella> prepareDevConfig`
```

Comment:
same as above
https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks